spec/acceptance/settings_spec.rb

```diff
@@ -51,6 +51,40 @@ class { 'pulpcore':
   end
 end
 
+describe 'PAGE_SIZE setting' do
+  context 'default PAGE_SIZE' do
```

Comment:
I'd keep this as a unit test, just to keep it fast. Acceptance tests are much slower and what are we really testing here? We already have tests for the default case. Testing with a custom page size can make sense, just to make sure the application doesn't crash on start up.

Reply:
Good point! I moved the default case into the unit tests, and left only the custom page size test in acceptance.